glsk/glsk-document-ucte/src/test/java/com/powsybl/glsk/ucte/UcteGlskValueProviderTest.java

```diff
@@ -179,7 +179,12 @@ public void testZeroLsk() {
         Network network = Importers.loadNetwork("testCase.xiidm", getClass().getResourceAsStream("/testCase.xiidm"));
         Instant instant = Instant.parse("2016-07-29T10:00:00Z");
         UcteGlskDocument ucteGlskDocument = UcteGlskDocument.importGlsk(getClass().getResourceAsStream("/TestZeroLsk.xml"));
-        ZonalData<LinearGlsk> ucteGlskProvider = ucteGlskDocument.getZonalGlsks(network, instant);
+        ZonalData<SensitivityVariableSet> ucteGlskProvider = ucteGlskDocument.getZonalGlsks(network, instant);
         assertNull(ucteGlskProvider.getData("10YFR-RTE------C"));
     }
+
+    //For readability purpose
+    private double findWeight(SensitivityVariableSet variables, String id) {
```

Comment:
Maybe it should be interesting to put this method in the sensitivity API?

Reply:
Yes but not this way. This is an inefficient way to go around the fact the SensitivityVariables are stored as a list and not indexed by id. We can discuss the idea to modify the api.